kumo-core/src/test/resources/log4j.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd" >
```

Comment:
I'd rather use logback instead of introduce log4j (Ignore if i was already using log4j, i don't recall)